pokete.py

```diff
@@ -1595,18 +1254,18 @@ def map_additions():
             # #                   ###########
             # #
             # #
-            ###""", ignore="#", ob_class=HightGrass,
+            ###""", ignore="#", ob_class=HighGrass,
                          ob_args=_map.poke_args, state="float")
     # adding
     _map.inner.add(_map, 0, 0)
 
     # playmap_21
-    _map = ob_maps["playmap_21"]
-    _map.dor_playmap_19 = Dor("_", state="float",
+    _map = obmp.ob_maps["playmap_21"]
+    _map.dor_playmap_19 = Door("_", state="float",
                               arg_proto={"map": "playmap_19",
                                          "x": 26, "y": 1})
-    _map.dor = DorToCenter()
```

Comment:
Due to changing `Dor` to `Door`, these lines require one further space of indentation:

```suggestion:
                               arg_proto={"map": "playmap_19",
                                          "x": 26, "y": 1})
```